app/routes/collections.all.tsx

```diff
@@ -0,0 +1,160 @@
+import {json, type LoaderFunctionArgs} from '@shopify/remix-oxygen';
+import {useLoaderData, Link, type MetaFunction} from '@remix-run/react';
+import {
+  Pagination,
+  getPaginationVariables,
+  Image,
+  Money,
+} from '@shopify/hydrogen';
+import type {ProductItemFragment} from 'storefrontapi.generated';
+import {useVariantUrl} from '~/lib/variants';
+
+export const meta: MetaFunction<typeof loader> = () => {
+  return [{title: `Hydrogen | Procuts`}];
```

Comment:
Is this supposed to say Hydrogen | Products?

Reply:
Hahaha, good one. I'll make a PR in their repo ;-)

Reply:
Thanks very much